USERGUIDE.md

```diff
@@ -130,6 +142,30 @@ Config.userDefaults?.set("MY_CUSTOM_ID", forKey: Config.Key.subscriptionKey.rawV
     * [MiniApp events](#miniapp-events)
     * [Load Mini app from Cache](#miniapp-load-cache)
 
+<a id="configure-mini-app"></a>
+
+### Configure MiniApp
+---
+
+1. Import the MiniApp SDK in your `UIApplicationDelegate`:
+
+```swift
+import MiniApp
+```
+
+2. `MiniApp.configure()` should be called at launch by `AppDelegate`. 
+This will ensure that Analytics will be loaded and the shared instance to be configured.
```

Comment:
I think we should have a more generalist message and don't talk about analytics there

Reply:
removed the second sentence 👍